sklearn/datasets/openml.py

```diff
@@ -424,17 +424,34 @@ def _get_data_qualities(data_id, data_home):
         return None
 
 
-def _get_data_shape(data_qualities):
-    # Using the data_info dictionary from _get_data_info_by_name to extract
-    # the number of samples / features
+def _get_data_instances(data_qualities):
+    """
+    Retrieve the number of instances (samples) from data qualities
+
+    Parameters
+    ----------
+    data_qualities : list
+        used to retrieve the number of instances (samples) in the dataset
+
+    Returns
+    -------
+    instances : int
+        the number of samples in the dataset or -1 if data qualities are
+        unavailable
+    """
+    # If the data qualities are not available, we return -1
+    default_instances = -1
+
     if data_qualities is None:
-        return None
+        return default_instances
+
     qualities = {d['name']: d['value'] for d in data_qualities}
     try:
-        return (int(float(qualities['NumberOfInstances'])),
-                int(float(qualities['NumberOfFeatures'])))
+        instances = int(float(qualities['NumberOfInstances']))
     except AttributeError:
```

Comment:
I don't see how an AttributeError would be raised here.

Reply:
I changed the way we get the value, so it's not needed anymore.
See https://github.com/scikit-learn/scikit-learn/pull/14623/files#diff-ea672b15dd808c88257c58681d17bb6aR448